### PR TITLE
fix(polyfills-loader): patch unpkg resolution error by specifying 'module'

### PR DESCRIPTION
**Pull request by atla5**
problem found by @daKmoR when bugfixing unpkg imports [[codepen](https://codepen.io/daKmoR/pen/ydeaQQ)]

### CODE REVIEW

File: packages/polyfills-loader/package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "@open-wc/polyfills-loader",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (daKmoR) [inline packages/polyfills-loader/package.json]: no need to bump version - CI takes care of it

```suggestion
  "version": "0.3.1",
```

**Comment by LarsDenBakker** [comment]
What are you using the polyfills loader for? We generally don't have it in any of our recommendations anymore.

**Comment by atla5** [comment]
I'm likely just following a previous recommendation. I've been working on a repo consuming it for some time now and didn't get the memo this was outmoded/unnecessary. 

is there anything replacing it? and if so, what's the new suggested makeup of the `main` entrypoint file collecting all of your web components?

**Comment by daKmoR** [comment]
still makes sense to fix it... but yeah we should deprecate it as well

**Comment by LarsDenBakker** [comment]
@atla5 we add the polyfills in the build, so you just write your code as you normally would.

Nothing wrong with the polyfills loader though, we just found a more efficient way to do it. You can take a look at `npm init @open-wc` and see how it looks and see if it works for you.

**Pull request merged**